### PR TITLE
Add filter for Closure functions

### PR DESCRIPTION
**Pull request by akoryak**
#### What does this PR do?
Add filter for Closure functions

#### Checklist
- [ x ] Pull request contains a clear definition of changes
- [ x ] Tests (either unit, integration, or acceptance) written and passing
- [ x ] Relevant documentation produced and/or updated

### CODE REVIEW

File: src/Filter/Closures.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    /**
+     * Filters $value to a Closure strictly.
+     *
+     * $value must be a bool or 'true' or 'false' disregarding case and whitespace.
+     *
+     * The return value is \Closure, as expected by the \TraderInteractive\Filterer class.
+     *
+     * @param mixed $value       the value to filter to a boolean
+     * @param bool  $allowNull   Set to true if NULL values are allowed. The filtered result of a NULL value is NULL
+     * @return bool|null the filtered $value
```
Comment (chrisryan) [inline src/Filter/Closures.php:21]:
```suggestion
     * @param bool  $allowNull   Set to true if NULL values are allowed. The filtered result of a NULL value is NULL
     *
     * @return bool|null the filtered $value
```

**Review by chadicus — COMMENTED** [review]
I'm not sure this is the right library for the closures filter class, but i'll defer to others.
Be sure to update the README.md file with instructions and usage examples

**Comment by akoryak** [comment]
> I'm not sure this is the right library for the closures filter class, but i'll defer to others. Be sure to update the README.md file with instructions and usage examples

1) updated the documentation
2) It doesn't have much sense to me to create new git repository with ~90% duplication for just one simple filter function. Second reason I why I added it here is that I don't see any use case for this closure filter be used standalone

**Review by chrisryan — DISMISSED** [review]

**Review by chrisryan — APPROVED** [review]

**Review by chrisryan — DISMISSED** [review]

File: src/Filter/Closures.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+class Closures
+{
+    /**
+     * Filters $value to a Closure strictly.
+     * The return value is \Closure, as expected by the \TraderInteractive\Filterer class.
+     *
+     * @param mixed $value       the value to filter to a closure function
+     * @param bool  $allowNull   Set to true if NULL values are allowed. The filtered result of a NULL value is NULL
+     *
+     * @return bool|null the filtered $value
```
Comment (chadicus) [inline src/Filter/Closures.php:19]:
```suggestion
     * @return Closure|null the filtered $value
```

File: src/Filter/Closures.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+class Closures
+{
+    /**
+     * Filters $value to a Closure strictly.
+     * The return value is \Closure, as expected by the \TraderInteractive\Filterer class.
+     *
+     * @param mixed $value       the value to filter to a closure function
+     * @param bool  $allowNull   Set to true if NULL values are allowed. The filtered result of a NULL value is NULL
+     *
+     * @return Closure|null the filtered $value
```
Comment (chadicus) [inline src/Filter/Closures.php:19]:
```suggestion
     * @return \Closure|null the filtered $value
```

**Review by chadicus — APPROVED** [review]

**Pull request merged**